src/vm/i386/excepx86.cpp

```diff
@@ -3731,4 +3752,10 @@ AdjustContextForVirtualStub(
     return TRUE;
 }
 
+#ifdef FEATURE_PAL
+VOID DECLSPEC_NORETURN DispatchManagedException(PAL_SEHException& ex, bool isHardwareException)
```

Comment:
The current definition of ``INSTALL_MANAGED_EXCEPTION_DISPATCHER`` and ``UNINSTALL_MANAGED_EXCEPTION_DISPATCHER`` uses ``DispatchManagedException`` even when ``WIN64EXCEPTIONS`` is not used:
```
299 #ifdef FEATURE_PAL
300 VOID DECLSPEC_NORETURN DispatchManagedException(PAL_SEHException& ex, bool isHardwareException);
301
302 #define INSTALL_MANAGED_EXCEPTION_DISPATCHER        \
303         PAL_SEHException exCopy;                    \
304         bool hasCaughtException = false;            \
305         try {
306
307 #define UNINSTALL_MANAGED_EXCEPTION_DISPATCHER      \
308         }                                           \
309         catch (PAL_SEHException& ex)                \
310         {                                           \
311             exCopy = std::move(ex);                 \
312             hasCaughtException = true;              \
313         }                                           \
314         if (hasCaughtException)                     \
315         {                                           \
316             DispatchManagedException(exCopy, false);\
317         }
318
```

For me, the use of ``PAL_SEHException`` seems to be related with ``FEATURE_PAL`` rather than ``WIN64EXCEPTIONS``, and thus I try to add its own ``DispatchManagedException`` instead of changing the above macros.

Please let me which one would be better: revise the macro or add a dummy dispatcher.